### PR TITLE
Remove slim from the docs

### PR DESCRIPTION
**Pull request by ignaciosy**
According to discussion [here](https://docs.google.com/document/d/1YzLqU95Bdxw8ORQ_5by3LASjHYUExrNtWQvZkEbSav8/edit#heading=h.6r6tke4nw7i8)

### CODE REVIEW

File: templates/bin/fastcheck [hunk 1]
```diff
@@ -19,8 +19,6 @@ bundle exec scss-lint `find app/assets/stylesheets/ -name "*.scss"`
 # new SASS linter
 yarn sass-lint -v --no-exit app/javascripts/**/*.scss
 
-bundle exec slim-lint app/views/ -c .slim-lint.yml
```
Comment (coorasse) [inline templates/bin/fastcheck:22]: can we have the erb alternative? or will you tackle that in a separate PR?

Comment (ignaciosy) [reply templates/bin/fastcheck:22]: aha the erb linter. Yes will add it here, thanks for the reminder

Comment (schmijos) [reply templates/bin/fastcheck:22]: Yes, please have a look at https://github.com/Shopify/erb-lint

**Review by schmijos — APPROVED** [review]

**Comment by coorasse** [comment]
@ignaciosy can we update and finish this PR?

**Review by coorasse — APPROVED** [review]

**Pull request merged**